vignettes/estimate_infections_workflow.Rmd

```diff
@@ -0,0 +1,259 @@
+See other vignettes for a more thorough exploration of [alternative model variants](estimate_infections_options.html) and [theoretical background](estimate_infections.html).
+
+# Data
+
+Obtaining a good and full understanding of the data being used an important first step in any inference procedure such as the one applied here. 
+_EpiNow2_ expects data in the format of a data frame with two columns, `date` and `confirm`, where `confirm` stands for the number of confirmed counts - although in reality this can be applied to any data including suspected cases and lab-confirmed outcomes.
+The user might already have the data as such a time series provided, for example, on public dashboards or directly from public health authorities.
+Alternatively, they can be constructed from individual-level data, for example using the [incidence2](https://cran.r-project.org/web/packages/incidence2/index.html) R package.
+An example data set called `example_confirm`  is included in the package:
+
+
+```r
+head(example_confirmed)
+#>          date confirm
+#> 1: 2020-02-22      14
+#> 2: 2020-02-23      62
+#> 3: 2020-02-24      53
+#> 4: 2020-02-25      97
+#> 5: 2020-02-26      93
+#> 6: 2020-02-27      78
+```
+
+Any estimation procedure is only as good as the data that feeds into it.
+A thorough understanding of the data that is used for _EpiNow2_ and its limitations is a prerequisite for its use.
+This includes but is not limited to biases in the population groups that are represented (_EpiNow2_ assumes a closed population with all infections being caused by other infections in the same population), reporting artefacts and delays, and completeness of reporting.
+Some of these can be mitigated using the routines available in _EpiNow2_ as described below, but others will cause biases in the results and need to be carefully considered when interpreting the results.
+
+# Set up
+
+We first load the _EpiNow2_ package.
+
+
+```r
+library("EpiNow2")
+```
+
+We then set the number of cores to use. We will want to run 4 MCMC chains in parallel so we set this to 4. If we had fewer than 4 available or wanted to run fewer than 4 chains (at the expense of some robustness), or had fewer than 4 computing cores available we could set it to that. To find out the number of cores available one can use the [detectCores](https://rdrr.io/r/parallel/detectCores.html) function from the `parallel` package.
+
+
+```r
+options(mc.cores = 4)
+```
+
+# Parameters
+
+Once a data set has been identified, a number of relevant parameters need to be considered before using _EpiNow2_.
+As these will affect any results, it is worth spending some time investigating what their values should be.
+
+## Delay distributions
+
+_EpiNow2_ works with different delays that apply to different parts of the infection and observation process.
+They are defined using a common interface with the `dist_spec()` function.
+For help with this function, see its manual page
```

Comment:
```suggestion
For help with this function, see its [manual page](https://epiforecasts.io/EpiNow2/dev/reference/dist_spec.html).
```

Reply:
Would be nice to be able to link but sticking with the original instead to ensure it's compatible with whatever version a user has.